Automattic/woocommerce-payments
# Display risk for payment methods without risk assessment

Fixes #2750 

#### Changes proposed in this Pull Request

Display risk for payment methods without risk assessment

Some UPE payment methods don't have risk assessed in every transaction. This is the case, for now, for P24, iDEAL and Bancontact, but probably there are other methods like SEPA payment without it as well.

In those cases the risk comes as `not_assessed`, and we were missing a mapping for that risk level.

This PR addresses that missing mapping, in both the Transactions table (also when downloading a CSV) and the Transaction summary view.

![image](https://user-images.githubusercontent.com/2612426/140081913-b46a9555-1515-48ea-9b6d-1728dcd5ae35.png)

![image](https://user-images.githubusercontent.com/2612426/140081777-df4f97f3-a58f-4ed2-9753-61d46a89955f.png)

![image](https://user-images.githubusercontent.com/2612426/140081658-14dca688-d125-43aa-a149-0d16699a9527.png)

#### Testing instructions

* [Enable P24](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-3.2.0#enable-p24-upe-method)
* Make a purchase paying with P24 (Needs to be using EUR or PLN for it to appear).
* Go to Payments - Transactions
* Click on the 3 dots in the upper right corner of the table and enable `Risk level` for the column to appear.
* Check the latest transaction shows `N/A` in the `Risk level` column instead of a blank space.
* Click on Download and check `N/A` is shown in the right column of the csv file.
* Click on the transaction
* Check `Risk evaluation` appears as `N/A` instead of `-`.

-------------------


**Post merge**
